### PR TITLE
Track 4 fields from mysql's information_schema.tables

### PR DESCRIPTION
**Pull request by nurblieh**
@sougou 
Fields: table_rows, data_length and index_length.
Doc: https://dev.mysql.com/doc/refman/5.6/en/tables-table.html

The goal is to export these variables for monitoring purposes.

I think data_free would also be useful but let me know how this CL looks OK before I go forward with that.

### CODE REVIEW

File: go/vt/tabletserver/schema_info.go [hunk 1]
```diff
@@ -185,6 +188,7 @@ func (si *SchemaInfo) Open(appParams, dbaParams *sqldb.ConnParams, schemaOverrid
 		if err != nil {
 			panic(PrefixTabletError(ErrFatal, err, fmt.Sprintf("Could not get load table %s: ", tableName)))
 		}
+		tableInfo.SetMysqlStats(row[4], row[5], row[6])
```
Comment (sougou) [inline go/vt/tabletserver/schema_info.go]: It's better to change NewTableInfo to require these parameters.

Comment (sougou) [inline go/vt/tabletserver/schema_info.go]: ok. nvm. I see why you need the separate funtion :).

**Comment by sougou** [comment]
Looking good so far.
Do you think you can also display this additional info in /schemaz?

**Comment by nurblieh** [comment]
On Tue, Aug 25, 2015 at 9:23 PM, sougou <notifications@github.com> wrote:

> Do you think you can also display this additional info in /schemaz?
>
Looks easy on the surface if I push the stats down into schema.Table
instead of table_info.TableInfo

But, I see that schema.Table is used in multiple places, like
vtgate/planbuilder/schema.go. Will have to see if that sullies other things.

**Comment by nurblieh** [comment]
Added data_free column. Useful for innodb.

**Comment by sougou** [comment]
LGTM
Looks like one of us will have to merge this. Let me know when everything's done.

**Pull request merged**